dev.env

```diff
@@ -1,8 +1,8 @@
 NODE_PATH=./src
 NEXT_PUBLIC_APR_ANALYTICS_ENDPOINT=https://analytics.ousd.com/api/v1/apr/trailing
 
-NEXT_PUBLIC_STATS_ENDPOINT=https://www.originprotocol.com
-NEXT_PUBLIC_STATS_ENDPOINT_LOCAL=http://localhost:5002
+NEXT_LEGACY_WEBSITE_HOST=https://www.originprotocol.com
```

Comment:
Let's make this https://api.originprotocol.com  - I just setup the cloudflare DNS and Heroku for it.